math.asm

```diff
@@ -187,8 +188,8 @@ ABS
     rts
 
 DABS_STAR           ; ( n1 n2 -- ud1 )
-    lda MSB,x	    ;	ud1 = abs(n1) * abs(n2)
-    eor MSB+1,x     ;	with final sign output in A register
+    lda MSB,x      ;   ud1 = abs(n1) * abs(n2)
+    eor MSB+1,x     ;  with final sign output in A register
```

Comment:
Couldn't get it right in the merge somehow :(

Reply:
it's ok!